### PR TITLE
[wip] wip on ci coverage fix

### PR DESCRIPTION
**Pull request by filipecosta90**
checking the gears CI file and making the AI one lookalike.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
             ./opt/readies/bin/getpy3
             BREW_NO_UPDATE=1 ./opt/system-setup.py
             # git clone git://github.com/antirez/redis.git --branch 5.0.7; (cd redis; make malloc=libc -j $(nproc); make install); redis-server --version
-            ./get_deps.sh cpu
-      - save_cache:
-          paths:
-            - deps
-            - /usr/local
-          key: build-dependencies-{{ checksum "get_deps.sh" }}
-      - run:
```
Comment (rafie) [inline .circleci/config.yml:27]: I''m not entirely sure the workspace setup code is redundant.

**Comment by lantiga** [comment]
@filipecosta90 is this still relevant after #342 has been merged?

**Comment by filipecosta90** [comment]
> @filipecosta90 is this still relevant after #342 has been merged?

we should discuss with @rafie further requirements on the test platforms like the platforms we discussed testing for:
- [x] buster
- [ ] stretch
- [ ] rhel7

what do you think Rafi?

**Comment by rafie** [comment]
@filipecosta90 It definitely wouldn't hurt to test on bionic and centos7/rhel7, which are the main deployment platforms in the cloud. This will also make us more confident with our builds being debian and not platform-specific. We should also find a way to automate testing of gpu builds in the cloud.
So the short answer is: yes. :-)

**Comment by filipecosta90** [comment]
closing in favor of newer build PRs

**Pull request closed without merging**